docs/how-to-guides/new-user-guides/helm-charts-in-rancher/oci-registries.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Using OCI Helm Chart Registries
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/helm-charts-in-rancher/oci-registries"/>
+</head>
+
+Helm v3 introduced storing Helm charts as [Open Container Initiative (OCI)](https://opencontainers.org/about/overview/) artifacts in container registries. With Rancher v2.9.0, you can add [OCI-based Helm chart registries](https://helm.sh/docs/topics/registries/) alongside http- and Git-based repositories. This means you can deploy apps that are stored as OCI artifacts.
+
+## Add an OCI Registry
+
+To add an OCI registry through the Rancher UI:
+
+1. Click **☰ > Cluster Management**.
+1. In the left navigation bar, under **Advanced**, select **Repositories**.
+1. Click **Create**.
+1. Enter a name and description for the registry. Select **OCI** as the target.
+1. Enter the URL for the registry. You can use this field to finetune how many charts you want to make availabe for installation on Rancher. More generic endpoints target more charts, as the following examples demonstrate:
+
+  1. **oci://\<registry-host\>/**: Every chart in the registry becomes available for installation, regardless of namespace or tag.
+  1. **oci://\<registry-host\>/\<namespace\>**: Every chart in the specified namespace within the registry becomes available for installation.
+  1. **oci://\<registry-host\>/\<namespace\>/\<chart-name\>**: Only the specified chart and any associated tags or versions of that chart become available for installation.
+  1. **oci://\<registry-host\>/\<namespace\>/\<chart-name\>:\<tag\>**: Only the chart with the specified tag becomes available for installation.
+
+1. Select **Basicauth** from the authentication field and enter a username and password as required. 
+1. Add any labels and annotations.
+1. Click **Create**.
+
+It may take some time for the OCI registry to activate. This is particularly true if the OCI endpoint contains multiple repositories. 
+
+To view Helm charts in the OCI registry after it achieves an `Active` state:
+
+1. Click **☰**. Under **Explore Cluster** in the left navigation menu, select a cluster.
+1. Click **Apps > Charts**.
+1. Select the OCI repository from the dropdown.
+
+The `spec.insecurePlainHttp` field allows insecure connections to OCI registries. When this field is set to `true`, Rancher will connect to the OCI endpoint without performing an SSL check. This works exactly the same as how the `spec.insecurePlainHttp` field works in [ORAS CLI](https://oras.land/docs/commands/use_oras_cli), since under the hood Rancher manager uses the ORAS library.
+
+The CRD that is linked to the OCI Helm registry is `ClusterRepo`.
+
+[Screenshot of the ClusterRepo YAML]<!-- Engineers - Can we get this screenshot? Thank you! -->
```

Comment:
I left this part in, but it feels like we need a little more content here to explain the relevance of mentioning it in this section vs somewhere else, and what users can use that information for.